src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -6668,30 +6668,13 @@ components:
           type: integer
         name:
           type: string
+        tableName:
+          type: string
+        columnName:
+          type: string
         kind:
           type: string
           enum: [ range, list ]
-        min:
-          type: integer
-        max:
-          type: integer
-        values:
-          type: array
-          items:
-            $ref: '#/components/schemas/SnapshotBuilderListOption'
-
-    SnapshotBuilderListOption:
-      type: object
-      description: >
```

Comment:
Why is ListOption being deleted as well?

Reply:
It is currently unused with this change. I think it is still necessary for your PR, so if yours merges first I'll update off your branch and add it back. (Otherwise I think if this merges first, probably what will happen is it will get readded by your change)